libkbfs/bserver_disk.go

```diff
@@ -123,6 +123,12 @@ func (b *BlockServerDisk) getStorage(tlfID tlf.ID) (
 func (b *BlockServerDisk) Get(ctx context.Context, tlfID tlf.ID, id kbfsblock.ID,
 	context kbfsblock.Context) (
 	data []byte, serverHalf kbfscrypto.BlockCryptKeyServerHalf, err error) {
+	select {
+	case <-ctx.Done():
+		return nil, kbfscrypto.BlockCryptKeyServerHalf{}, ctx.Err()
```

Comment:
Can you do `errors.WithStack(ctx.Err())` so tests would show what timed out?

This can probably be decomped out to a utility function somewhere:

```
func checkContext(ctx context.Context) error {
  select {
    case <-ctx.Done():
      return errors.WithStack(ctx.Err())
    default:
      return nil
  }
}
```

and all the call sites would look like:

```
err := checkContext(ctx)
if err != nil {
  return ..., err
}
```